gateway/cert_test.go

```diff
@@ -1384,8 +1384,6 @@ func TestAPICertificate(t *testing.T) {
 }
 
 func TestCertificateHandlerTLS(t *testing.T) {
-	test.Flaky(t) // TODO: TT-5261
```

Comment:
Did you do any stress tests on this, something like -count=100...1000?

Reply:
yes, essentially the fix (hacky) is to start fresh, delete all certs before starting the test.